### DISCUSSION TITLE
How to get individual player data ?

### DISCUSSION THREAD

**Discussion by The0xman** [cwendt94/espn-api#533] [ESPN API]
Hello, I am a little bit confused on how to use this api. I have connected to the API for my league and have all the data provided from the League class, but I am not sure how I go about getting individual player data…

That is, how do I get the data that I should pass as the data parameter in the Player class.

**Comment by cwendt94** [comment]
You don't have to manually initiate the player class, it will automatically get filled out through the [League interface](https://github.com/cwendt94/espn-api/wiki/League-Class). Depending on what type of data, it could already be filled out. If you want to check out players on a team you can access it this way
```python
>>> team = league.teams[0]
>>> player = team.roster[0]
>>> player
Player(Travis Kelce)
>>> player.name
'Travis Kelce'
>>> player.playerId
15847
>>> player.posRank
10
>>> player.position
'TE'
>>> player.proTeam
'KC'
>>> player.eligibleSlots
['WR/TE', 'TE', 'OP', 'BE', 'IR', 'RB/WR/TE']
>>> player.acquisitionType
'DRAFT'
```

Here is the [football intro wiki page](https://github.com/cwendt94/espn-api/wiki/Football-Intro) that goes over some of the different features. On the right hand side of the wiki page it also describes the different fields in the classes.
